scripts/lib/installer-win32.tsx

```diff
@@ -43,7 +43,7 @@ function getAppVersion(appDir: string): string {
   return offset ? `${ semver }.${ offset }` : semver;
 }
 
-async function buildCustomAction(): Promise<void> {
+export async function buildCustomAction(): Promise<void> {
```

Comment:
I assume this function is in this file and not in `package.ts` just to keep the latter smaller, although it used to be called locally.

Reply:
That, and it's related to building the installer.